specfile/specfile.py

```diff
@@ -50,9 +65,13 @@ class Specfile:
         autosave: Whether to automatically save any changes made.
     """
 
+    ENCODING_ARGS = {"encoding": "utf8", "errors": "surrogateescape"}
```

Comment:
To stop mypy from complaining about this, it is necessary to assign types to specific keys (`TypedDict` is from `typing`):

`EncodingArgs = TypedDict("EncodingArgs", {"encoding": str, "errors": str})`

And then use the type:

```suggestion
    ENCODING_ARGS: EncodingArgs = {"encoding": "utf8", "errors": "surrogateescape"}
```